tekton/src/pipelines/taskdefs/fvt-manage/phase4.yml.j2

```diff
@@ -135,3 +135,44 @@
     - fvt-manage-esig-basic
     - fvt-manage-eaudit-basic
     - fvt-manage-mif-kafka-setup
+
+# Manage FVT BIRT Reports
+- name: fvt-manage-birt-report
+  {{ lookup('template', 'taskdefs/fvt-manage/ui/taskref.yml.j2') | indent(2) }}
+  {{ lookup('template', 'taskdefs/fvt-manage/ui/when.yml.j2') | indent(2) }}
+  params:
+    {{ lookup('template', 'taskdefs/fvt-manage/ui/params.yml.j2') | indent(4) }}
+    - name: fvt_test_suite_prefix
+      value: core
+    - name: fvt_test_suite
+      value: birtreportexe
+  runAfter:
+    - fvt-manage-wo-basic
+    - fvt-manage-escalation-action
+    - fvt-manage-autoscript-adddeletewo
+    - fvt-manage-jms-inbound
+    - fvt-manage-esig-basic
+    - fvt-manage-eaudit-basic
+    - fvt-manage-mif-kafka-setup
+    - fvt-manage-communication-temp
+
+# Manage FVT Adhoc Reports
+- name: fvt-manage-adhoc-report
```

Comment:
This task needs to be added to phase5 runafter ->https://github.com/ibm-mas/cli/blob/squadci/tekton/src/pipelines/taskdefs/fvt-manage/phase5.yml.j2#L17-L22

Reply:
I made the changes and kept adhoc and birt report tests before the last phase. Is that okay? Let me know if this does not look good.

Thanks!

Reply:
I have pushed additional changes and tasks now look better. Slack thread where this was discussed
https://ibm-watson-iot.slack.com/archives/C04UR4PPCRY/p1696339457555709